### PR TITLE
Blog App: Added data model

### PR DESCRIPTION
**Pull request by amare1990**
- [ ] Setup Gitflow for the project.
- [ ] Added and configured `ror` linters.
- [ ] Created migration file for the corresponding entity in the ERD from the project requirement.
- [ ] Run the migrations to get the database schema.
- [ ] Checked linters errors.

### CODE REVIEW

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @amare1990, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please know that you should have a migration file for creating `comments` table, kindly add that.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230124221753_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    create_table :posts do |t|
+      t.integer :authorid
+      t.string :title
+      t.text :text
+      t.integer :commentscounter
+      t.integer :likescounter
+
+      t.timestamps
+    end
+    add_index :posts, :authorid
```
Comment (rloterh) [inline db/migrate/20230124221753_create_posts.rb:12]: - [ ] OPTIONAL
Taking into consideration the given ERD, we can see that `user` model is referenced using foreign key in models as `AuthorId`. Let's therefore bear in mind that `author` is aliasing the `user` model, and hence, our migration query should be similar to:

```ruby
...
  def change
    add_reference :posts, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...

```

or.. 

```ruby
  create_table :posts do |t|
    . . .
    t.references :author, foreign_key: { to_table: 'users' } 
    . . .
  end
```

File: db/migrate/20230124222320_create_likes.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :authorid
+      t.integer :postid
+
+      t.timestamps
+    end
+    add_index :likes, :authorid
```
Comment (rloterh) [inline db/migrate/20230124222320_create_likes.rb:9]: - [ ]OPTIONAL
 Similarly, the `user` model is referenced using foreign key in models as `AuthorId`, and `author` an alias of `user` hence, our migration query should be similar to:

```ruby
...
  def change
    add_reference :likes, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...

```

or.. 

```ruby
  create_table :likes do |t|
    . . .
    t.references :author, foreign_key: { to_table: 'users' } 
    . . .
  end
```

File: db/migrate/20230125060440_create_comments.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :authorid
+      t.integer :postid
+      t.text :text
+
+      t.timestamps
+    end
+    add_index :comments, :authorid
```
Comment (rloterh) [inline db/migrate/20230125060440_create_comments.rb:10]: - [ ] Again, the `user` model is referenced using foreign key in models as `AuthorId`, and `author` an alias of `user` hence, our migration query should be similar to:

```ruby
...
  def change
    add_reference :comments, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...

```

or.. 

```ruby
  create_table :comments do |t|
    . . .
    t.references :author, foreign_key: { to_table: 'users' } 
    . . .
  end
```

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Amare,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Good job with migration query!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

**You can also consider:**
- MIT license link in your ReadMe is broken, kindly endeavor to fix it.

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**